templates/devx-experimental/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extend": "base",
+  "devDependencies": ["fl-exp"],
```

Comment:
This is published to npm?

Reply:
Yeah although this is just a placeholder. I assume we'd be using a different name when/if we decide this is the right direction.